src/frontend/screens/Onboarding/OnboardingPrivacyPolicy.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import {
+  View,
+  Text,
+  ScrollView,
+  StyleSheet,
+  TouchableOpacity,
+} from 'react-native';
+import {useIntl, defineMessages} from 'react-intl';
+import {PrivacyPolicy} from '../PrivacyPolicy';
+import {BLUE_GREY, WHITE, BLACK, COMAPEO_BLUE} from '../../lib/styles';
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
+
+import {AppDiagnosticMetrics} from '../../metrics/AppDiagnosticMetrics';
+import {DeviceDiagnosticMetrics} from '../../metrics/DeviceDiagnosticMetrics';
+
+const m = defineMessages({
+  navTitle: {
+    id: 'screens.OnboardingPrivacyPolicy.navTitle',
+    defaultMessage: 'Privacy Policy',
+  },
+  permissionsTitle: {
+    id: 'screens.OnboardingPrivacyPolicy.permissionsTitle',
+    defaultMessage: 'Current Permissions',
+  },
+  shareDiagnostics: {
+    id: 'screens.OnboardingPrivacyPolicy.shareDiagnostics',
+    defaultMessage: 'Share Diagnostic Information',
+  },
+});
+
+export const OnboardingPrivacyPolicy = () => {
+  const appDiagnosticMetrics = new AppDiagnosticMetrics();
+  const deviceDiagnosticMetrics = new DeviceDiagnosticMetrics();
```

Comment:
these shouldn't be instantiated here. see comment in `App.tsx`. 

```suggestion
  const appDiagnosticMetrics = new AppDiagnosticMetrics();
  const deviceDiagnosticMetrics = new DeviceDiagnosticMetrics();
```

if you need access to the instances in this component, you can either:

1. import the instance(s) created in `App.tsx`
2. create a app-level context + hook and use the hook in this component (see `ApiContext` as an example

I think (2) is preferable, mostly for reasons that don't fully apply to our codebase just yet (testing)